liquid-base/liquid-base.cabal

```diff
@@ -97,7 +97,7 @@ library
                     Liquid.Prelude.NotReal
                     Prelude
   hs-source-dirs:     src
-  build-depends:      base                 >= 4.11.1.0 && < 5
+  build-depends:      base                 >= 4.14.0.0 && < 5
                     , liquid-ghc-prim
                     , liquidhaskell        >= 0.8.10.1
   default-language:   Haskell98
```

Comment:
Why `Haskell98` and not `Haskell2010`?

Reply:
I think this was a copy/paste of some other template I had lying around. I see no reason why we can't use `Haskell2010` here.